### PR TITLE
SISRP-36756 - Final Exams Schedule (Students)

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.berkeley.edu/browse/SISRP-36756

### CODE REVIEW

**Comment by redconfetti** [comment]
I just tried to troubleshoot the failing test error. It appears that even after trying to insert the date/time into the H2 database, it still comes out converted by the local time offset, even though it's being outputted as a UTC `Time` object.

**Comment by redconfetti** [comment]
Various errors concerning missing database tables are showing up:
```
.[2018-07-17 17:52:14] [ERROR] [EdoOracle::Queries] Query failed: ActiveRecord::StatementInvalid: ActiveRecord::JDBCError: org.h2.jdbc.JdbcSQLException: Table "CLC_TERMV00_VW" not found; SQL statement:
        SELECT ACADCAREER_CODE as career_code,
          TERM_ID as term_id,
          TERM_TYPE as term_type,
          TERM_YEAR as term_year,
          TERM_CODE as term_code,
          TERM_DESCR as term_descr,
          TERM_BEGIN_DT as term_begin_date,
          TERM_END_DT as term_end_date,
          CLASS_BEGIN_DT as class_begin_date,
          CLASS_END_DT as class_end_date,
          INSTRUCTION_END_DT as instruction_end_date,
          GRADES_ENTERED_DT as grades_entered_date,
          FINAL_EXAM_WK_BEGIN_DT as final_exam_week_start_date,
          END_DROP_ADD_DT as end_drop_add_date,
          IS_SUMMER as is_summer
        FROM  SISEDO.CLC_TERMV00_VW
        WHERE INSTITUTION = 'UCB01' AND
          ACADCAREER_CODE = 'UGRD' AND
          TERM_TYPE IS NOT NULL
        ORDER BY TERM_ID DESC
```

I'm creating entries for those in `populate_sisedo_h2.rb`.

File: app/models/edo_oracle/queries.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Queries < Connection
       TRIM(crs."transcriptTitle") AS course_title_short,
       crs."subjectArea" AS dept_name,
       crs."classSubjectArea" AS dept_code,
+      crs."academicCareer-code" AS course_career_code,
```
Comment (lyttam) [inline app/models/edo_oracle/queries.rb]: @redconfetti Does this column really represent the course career? "Course career" has a specific meaning in Campus Solutions that is separate from "academic career".  Unless the column is being populated from the `CRSE_CAREER` field on the underlying PS table, I suggest renaming it `academic_career_code`.

Comment (redconfetti) [reply app/models/edo_oracle/queries.rb]: @lyttam  - Yes, it is the CRSE_CAREER code.

**Pull request merged**